src/DoneList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import DoneItem from './DoneItem';
+
+const DoneListBlock = styled.div`
+    background: white;
+    width: 300px;
+    height: 250px;
+    overflow-x: scroll;
+`
+
+const DoneList = (props) => {
+    const {onChangeTodo,onRemove,dones}=props;
```

Comment:
```suggestion
const DoneList = ({onChangeTodo, onRemove, dones}) => {
```
구조분해할당으로 바로 props들을 받아올 수 있습니다!!

Reply:
아아 props로 받아와서 다시 변수를 선언하지 않아도 되네요!!